integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionGlobalSortTestCase.scala

```diff
@@ -939,7 +939,7 @@ class StandardPartitionGlobalSortTestCase extends QueryTest with BeforeAndAfterA
     assert(exMessage.getMessage.contains("day is not a valid partition column in table default.partitionnocolumn"))
   }
 
-  ignore("data loading with default partition in static partition table with batchsort") {
+  ignore("data loading with default partition in static partition table with local_sort") {
     sql("DROP TABLE IF EXISTS partitiondefaultbatchsort")
     sql(
       """
```

Comment:
Table name is still a batch sort, better to rename for readability.
looked for "batchsort" in the code. only this file and TestCreateTableWithSortScope has this problem